eclipse-theia/theia
# Use app shell methods for expand / collapse for toggle terminal command

#### What it does

Fixes the `toggleTerminal()` method to use the app shell expand/collapse methods.

Resolves #13129 

#### How to test

1. Run `yarn electron build` and then `yarn electron start`
2. Press ctrl+\` to trigger the "Terminal: Toggle Terminal" command
3. The bottom panel should animate showing / hiding, this didn't happen before because the bottom panel was directly being shown/hidden

![image](https://github.com/eclipse-theia/theia/assets/146364035/d6fbfdfd-485d-491a-a125-dc270dcfdc74)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
